### PR TITLE
sys/arm64: add CPU part identifiers for Apple M1 and M2

### PR DESCRIPTION
**Pull request by llfw**
Obtained from:	Linux

### CODE REVIEW

File: sys/arm64/arm64/identcpu.c [hunk 1]
```diff
@@ -250,6 +250,23 @@ static const struct cpu_parts cpu_parts_qcom[] = {
...
+	{ CPU_PART_M1_FIRESTORM_PRO, "M1 Pro Firestorm" },
+	{ CPU_PART_M1_ICESTORM_MAX, "M1 Max Icestorm" },
+	{ CPU_PART_M1_FIRESTORM_MAX, "M1 Max Firestorm" },
+	{ CPU_PART_M2_BLIZZARD, "M2 Blizzard" },
+	{ CPU_PART_M2_AVALANCHE, "M2 Avalanche" },
+	{ CPU_PART_M2_BLIZZARD_PRO, "M2 Pro Blizzard" },
+	{ CPU_PART_M2_AVALANCHE_PRO, "M2 Pro Avalanche" },
+	{ CPU_PART_M2_BLIZZARD_MAX, "M2 Max Blizzard" },
+	{ CPU_PART_M2_AVALANCHE_MAX, "M2 Max Avalanche" },
+	CPU_PART_NONE,
```
Comment (bsdimp) [inline sys/arm64/arm64/identcpu.c:267]: Do we have M3 part numbers?
If so, I can test this on my notebook..

**Comment by bsdimp** [comment]
What's there looks good to me...  
Not sure of the commit message, though...   The data was obtained there, but this code wasn't cut and pasted there.

**Comment by llfw** [comment]
i'll see if i can find M3 IDs (maybe OpenBSD has them) and update the commit message at the same time - would "Source:" be better than "Obtained from"?

**Comment by llfw** [comment]
i couldn't find anyone who has part numbers for the M3 CPUs and unfortuantely ran out of tuits for this relatively minor issue :-)  hopefully, people who own M3 hardware can submit the IDs for their CPUs.

i updated the commit message to be a bit more clear.

**Comment by bsdimp** [comment]
Source is better
And this is complete enough without them if there is any hassle for m3.

**Comment by markjdb** [comment]
Thanks, committed in da2b732288c72b347b128e1fa9a169c7bba3beb8.

**Pull request closed without merging**